src/lib/Api.js

```diff
@@ -49,12 +50,11 @@ class Api {
     return this;
   }
 
-  projectToken = () => {
+  projectTokenAndUrl = () => {
     return getInstance().then(d2 => {
-      return d2.dataStore.get("blsq-dataviz").then(namespace => {
-        console.log(namespace);
-        return namespace.get(window.tokenName).then(value => {
-          return value["token"];
+      return d2.dataStore.get("hesabu").then(namespace => {
```

Comment:
Stupid question probably, but do we need al the `return`s here? (My JS-fu isn't strong enough to know if they make a difference here, we need them to get all the promises to resolve?)

Reply:
(Each promise returns a new promise, so the top one knows to wait for it)